pgt/src/PGT/Output/Test.hs

```diff
@@ -31,35 +32,47 @@ data Test' commentary queryPlan = Test
   { commentary :: commentary
   , queryPlan  :: Maybe queryPlan
   , result     :: Result
+  , subTests   :: [Test' Comment queryPlan]
```

Comment:
Comment: subTests represent auxiliary test results that are different from the main test but still part of the same setup.